packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -352,6 +352,12 @@ export async function switchChain({
 
     onSuccess?.()
   } catch (err: any) {
+    // User cancelled the switch
+    if (err.code === 4001) {
```

Comment:
We have a lot of similar checks in the codebase, so let's extract a utility function we can use:

```
function isUserRejectedError(error: any) {
  return error.code === 4001 || error.code === 'ACTION_REJECTED';
}
```

Can't remember exactly where `ACTION_REJECTED` comes from, I think it might be Frame

Reply:
Changed. (`ACTION_REJECTED` actually come from etherjs)